mw-webapp/src/service/UserService.ts

```diff
@@ -27,6 +38,22 @@ export class UserService {
     return validatedUsersDTO;
   }
 
+  /**
+   * Get UsersDTO by uuids
+   */
+  public static async getUsersDTOByUuids(userUuids: string[]): Promise<UserDTO[]> {
+    const usersRef = collection(db, PATH_TO_USERS_COLLECTION);
+    const usersFilteredByUuid = query(usersRef, where(USER_UUID_FIELD, "in", userUuids));
+    const usersRaw = await getDocs(usersFilteredByUuid);
+    const usersDTO = querySnapshotToDTOConverter<UserDTO>(usersRaw);
+
```

Comment:
lets use approach which help us to receive more than 30 entities like in getSortedDayReportsDTO, but we need to build general helper, check it

Reply:
Copied approach